README.md

```diff
@@ -52,18 +52,13 @@ It also guarantees that the design is sound before code is written; a GitHub pul
 Typos and grammatical errors can go straight to a pull-request.
 When in doubt, start on the [mailing-list](#mailing-list).
 
-### Weekly Call
-
-The contributors and maintainers of all OCI projects have a weekly meeting on Wednesdays at:
-
-* 8:00 AM (USA Pacific), during [odd weeks][iso-week].
-* 2:00 PM (USA Pacific), during [even weeks][iso-week].
+### Meetings
 
+The contributors and maintainers of all OCI projects have monthly meetings at 5:00 PM (USA Pacific) on the first Wednesday of every month.
 There is an [iCalendar][rfc5545] format for the meetings [here](meeting.ics).
-
 Everyone is welcome to participate via [UberConference web][uberconference] or audio-only: +1 415 968 0849 (no PIN needed).
-An initial agenda will be posted to the [mailing list](#mailing-list) earlier in the week, and everyone is welcome to propose additional topics or suggest other agenda alterations there.
-Minutes are posted to the [mailing list](#mailing-list) and minutes from past calls are archived [here][minutes], with minutes from especially old meetings (September 2015 and earlier) archived [here][runtime-wiki].
```

Comment:
> I thought the mailing-list postings were useful IMO, as it reminded people that the meeting took place. And since the meetings are going to be monthly, it's less effort.

I think the agenda thread is enough of a reminder, but whatever, I've dropped that commit from this PR with 1077f05 → f6f2514.  I'll stop being the backstop minute-link poster, and folks who find the minute-link post useful can pick it up on their own (@vbatts posts minute links occasionally already).